packages/formation/sass/modules/_m-promo-banner.scss

```diff
@@ -0,0 +1,55 @@
+.vads-c-promo-banner {
+  z-index: $base-layer + 1;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+
+  background-color: $color-gray-lightest;
+  color: $color-link-default;
+  font-weight: 700;
+
+  .vads-c-promo-banner__body {
```

Comment:
Entirely optionally but I think it's a bit more SASS-y (heh heh) to write these nested classes with the `&` so we aren't rewriting the same class name. So `.vads-c-promo-banner__body` -> `&__body` I wonder if anyone else has strong feelings about this one way or another.

Reply:
@erikphansen It is, but it makes code a lot harder to search. I've run into lots of issues trying to find things myself. Which is why I added this rule: https://design.va.gov/documentation/developers#searchable-selectors